bib/cmd/bootc-image-builder/main.go

```diff
@@ -210,7 +212,7 @@ func manifestFromCobra(cmd *cobra.Command, args []string) ([]byte, error) {
 
 	manifestConfig := &ManifestConfig{
 		Imgref:       imgref,
-		ImgType:      imgType,
+		ImgType:      imgTypes[0],
```

Comment:
I am not an osbuild expert but I think we need a distinct manifest for each image.

Reply:
Yes I wanted to ask that, I also know very little about it.

Reply:
From looking at osbuild it says you can specify --export XYZ multiple times, for one manifest. I might be mistaken, and need to call osbuild multiple times ones for each diskimage type. and create multiple manifest.  The documentation is not very clear. Definitely do not want to be pulling the image multiple times from the registry.

Reply:
You're right, you can export the result of multiple pipelines.  The ISO build is too different to share any meaningful state with the disk images, but the AMI and QCOW2 basically share the whole manifest, and the QCOW2 has an extra pipeline at the end to convert the image format.  So we can build one manifest and export two artifacts, the `image` and the `qcow2`, and get two outputs from one build job.

Reply:
Why is it called image and not ami, raw, hyperv, qcow2 ...

As we roll this forward I would prefer the name of the directories and content to match the specified --type?

Reply:
> Why is it called image and not ami, raw, hyperv, qcow2 ...

That's just the name of the internal pipeline as it's named in the manifest.  It's plain `image` because the ami is just a raw disk so the pipeline name is generic.

Reply:
I want the images that are output named to match the image name or let me name the image. I was thinking that 
`BIB quay.io/podman/foobar --type ami --type raw --type hperv --type qcow` I would end up with a directry tree of 

raw/roobar.raw
ami/foobar.ami
hyperv/foobar.hyperv
qcow2/foobar.qcow2

OR


image/roobar.raw
image/foobar.ami
image/foobar.hyperv
image/foobar.qcow2

From a user perspective, I like the second one best, but either can be worked with.

Reply:
This can all be handled trivially from inside the code, so nothing serious to block on right now.  Let's do it in a small follow-up.

To clarify, my comments above were descriptions about internal naming which matter in the code, not suggestions for external-facing things like flags or directory names.

Putting everything in the same subdirectory of the output path with appropriate extensions makes total sense.  Also, naming the files after the container name is a nice idea, let's do that (again another follow-up).  I'll open an issue so we don't lose the ideas.